demo/JENKINS_HOME/jobs/docker-workflow/config.xml

```diff
@@ -50,13 +53,17 @@ docker.withRegistry(&apos;https://docker.example.com/&apos;, &apos;docker-regist
     stage &apos;Test Image&apos;
     // Run the petclinic app in its own docker container
     pcImg.withRun {petclinic -&gt;
-        // Spin up a maven test container, linking it to the petclinic app container allowing
-        // the maven tests to fire HTTP requests between the containers.
-        maven32.inside(&quot;-v /m2repo:/m2repo --link=${petclinic.id}:petclinic&quot;) {
-            git &apos;https://github.com/tfennelly/spring-petclinic-tests.git&apos;
+        dir(&apos;test&apos;) {
+        git url: &apos;https://github.com/jglick/spring-petclinic-tests.git&apos;, branch: &apos;selenium&apos;
 
+        // Spin up a maven + xvnc test container, linking it to the petclinic app container allowing
+        // the maven tests to fire HTTP requests between the containers.
+        docker.build(&apos;examplecorp/spring-petclinic-tests:snapshot&apos;).inside(&quot;-v /m2repo:/m2repo --link=${petclinic.id}:petclinic&quot;) {
             writeFile file: &apos;settings.xml&apos;, text: &apos;&lt;settings&gt;&lt;localRepository&gt;/m2repo&lt;/localRepository&gt;&lt;/settings&gt;&apos;
-            sh &apos;mvn -B -s settings.xml clean package&apos;
+            wrap([$class: &apos;Xvnc&apos;, takeScreenshot: false, useXauthority: true]) {
+              sh &apos;mvn -B -s settings.xml clean package&apos;
+            }
```

Comment:
Might be useful to add a comment with a link to the [build wrapper docs](https://github.com/jenkinsci/workflow-plugin/blob/master/basic-steps/CORE-STEPS.md#build-wrappers)?